modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp

```diff
@@ -0,0 +1,123 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+String className = (String)request.getAttribute("contact_information.jsp-className");
+long classPK = (long)request.getAttribute("contact_information.jsp-classPK");
+
+String emptyResultsMessage = ParamUtil.getString(request, "emptyResultsMessage");
+
+List<Address> addresses = AddressServiceUtil.getAddresses(className, classPK);
```

Comment:
This is wrong because with this we are not able to separate the already existing addresses and the new "contact addresses". But we can adjust this with the related story

Reply:
@smottal ic, so are you saything the class name we should be passing is "Contact.class.getName()" and the classPK should be contactId?

Reply:
Left as is for now.